cmd/earthly/main.go

```diff
@@ -1087,21 +1085,12 @@ type addresses struct {
 	localRegistry string
 }
 
-func (app *earthlyApp) getAddressesOriginal() (addresses, error) {
-	dbAddr := fmt.Sprintf("tcp://127.0.0.1:%v", app.cfg.Global.DebuggerPort)
-	return addresses{
-		buildkit:      app.buildkitHost,
-		debugger:      dbAddr,
-		localRegistry: app.cfg.Global.LocalRegistryHost,
-	}, nil
-}
-
-func (app *earthlyApp) getAddressesForTCP(context *cli.Context) (addresses, error) {
+func (app *earthlyApp) getAndValidateAddresses(context *cli.Context) (addresses, error) {
 	if !context.IsSet("buildkit-host") {
 		if app.cfg.Global.BuildkitHost != "" {
 			app.buildkitHost = app.cfg.Global.BuildkitHost
 		} else {
-			app.buildkitHost = buildkitd.TCPAddress
+			app.buildkitHost = buildkitd.DockerAddress
```

Comment:
In keeping with this as our default address scheme, because this is the only codepath now.